src/frequently-requested-changes.md

```diff
@@ -214,6 +214,29 @@ of tail or internal padding, as they can be reused due to the lack of a possible
 
 Cross-referencing to other discussions:
 
-* https://github.com/rust-lang/rfcs/issues/1397
-* https://github.com/rust-lang/rust/issues/17027
-* https://github.com/rust-lang/unsafe-code-guidelines/issues/176
+* <https://github.com/rust-lang/rfcs/issues/1397>
+* <https://github.com/rust-lang/rust/issues/17027>
+* <https://github.com/rust-lang/unsafe-code-guidelines/issues/176>
+
+## A way to bypass visibility, including an `unsafe` bypass
+
+Items are only accessible if they are marked `pub` or re-exported as such;
```

Comment:
This line doesn't read correctly to me as stated, i.e. "or re-exported as such" seems too strong.  E.g.:

```rust
mod m {
    fn f() {}
}
pub use m::f; //~ ERROR function `f` is private
```

Reply:
I see, I was under the impression you could do something like that. I agree it doesn't make sense as written.